README.md

```diff
@@ -97,8 +97,9 @@ Use zeno-ghq-cd zle
 # if setting fzf-tmux options
 # export ZENO_FZF_TMUX_OPTIONS="-p"
 
-# Experimental: Use UNIX Domain Socket
-export ZENO_ENABLE_SOCK=1
+# Recommended: Use UNIX Domain Socket
+# if disable
+# export ZENO_ENABLE_SOCK=0
```

Comment:
I suggest changing the variable.
Expecting default values to be empty is a good design for zero configuration.

```suggestion
# by default a unix domain socket is used
# if disable it
# export ZENO_DISABLE_SOCK=1
```

Also change the line in file *shell/function/zeno-call-client-and-fallback*.

```diff
--- a/shell/function/zeno-call-client-and-fallback
+++ b/shell/function/zeno-call-client-and-fallback
@@ -1,6 +1,6 @@
 #autoload
 
-if [[ -n $ZENO_ENABLE_SOCK ]]; then
+if [[ -z $ZENO_DISABLE_SOCK ]]; then
   (( $+functions[zeno-client] )) || zeno-enable-sock
   if [[ -S $ZENO_SOCK ]]; then
     zeno-client "$@"
```